target/linux/ramips/base-files/lib/upgrade/platform.sh

```diff
@@ -200,7 +200,8 @@ platform_check_image() {
 	zbt-wg3526-32M|\
 	zbt-wr8305rt|\
 	zte-q7|\
-	youku-yk1)
+	youku-yk1|\
+ atel-u270)
```

Comment:
Alphabetical order and missing indent. Please make sure to use tabs for indent instead of spaces.